tests/acceptance/reminder-list-test.js

```diff
@@ -29,3 +28,36 @@ test('clicking on an individual item', function(assert) {
     assert.equal(Ember.$('.reminder-item:first').text().trim(), Ember.$('.reminder-title').text().trim());
   });
 });
+
+test('clicking add new reminder reroutes to reminder/new', function(assert) {
+  server.createList('reminder', 5);
+
+  visit('/');
+  click('.create');
+
+
+  andThen(function() {
+    assert.equal(currentURL(), '/reminders/new');
+    assert.equal(Ember.$('input[type="text"]').length, 4);
+  });
+});
+
+test('user can create a new reminder', function(assert) {
+  server.createList('reminder', 5);
+
+  visit('/');
+  click('.create');
+
+  // andThen(function() {
+  //   assert.equal(Ember.$('.reminder-item').length, 5);
```

Comment:
So either keep this in or take it out. It's up to you. Personally I like that you're confirming that the length is 5 before you add one and confirm it's six.